### DISCUSSION TITLE
[Question] I have a question about verbosity

### DISCUSSION THREAD

**Discussion by Darkcast** [dwisiswant0/apkleaks#59] [Q&A]
Hi

is there a way to add more verbosity to the can of an apk. 
for example the scan will tell me i found xyz but it doesnt tell me in which file and it would be benefitials to inspect the file to figure out what else might be hidding in there. 

For example:

[Google_API_Key]
- AAAAABBBBCCCDDDEEEEFFF
| --> /resources/some_folder/somefile.properties

Thanks

**Comment by dwisiswant0** [comment]
This is a good idea we can improve.
